common/src/main/java/fi/hel/verkkokauppa/common/rest/CommonServiceConfigurationClient.java

```diff
@@ -134,6 +136,36 @@ public String getMerchantConfigurationValue(String merchantId, String namespace,
         }
     }
 
```

Comment:
Created this update merchant config value for functional tests. Mainly to make tests that verify the 403 return if the MERCHANT_PAYTRAIL_MERCHANT_ID or paytrail merchant secret key is not set.